src/context/reducers/account.ts

```diff
@@ -0,0 +1,45 @@
+import StellarSdk from "stellar-sdk";
```

Comment:
Just import the methods that we use, do not import the whole dependency, please.